release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -25,7 +25,7 @@ class TestScenario(enum.Enum):
 
 if os.environ.get("RAY_IMAGE") is not None:
     ray_image = os.environ.get("RAY_IMAGE")
-elif ray.__version__ != "3.0.0.dev0":
+elif ray.__version__ != "2.8.0":
```

Comment:
this should not be changed. it is using "3.0.0.dev" to check if it is on a release branch.